main.py

```diff
@@ -123,14 +123,19 @@ def add_seed():
 @app.route('/seeds/<IMSI>', methods=['GET'])
 def get_seed_msisdn(IMSI):
     try:
-        db_path = f".db/{IMSI}.db"
-        if os.path.exists(db_path):
-            con = sqlite3.connect(db_path)
+        dirname = os.path.dirname(__file__)
+        db_file = f".db/{IMSI}.db"
+        filename = os.path.join(dirname, db_file)
+        if os.path.exists(filename):
+            con = sqlite3.connect(filename)
             cur = con.cursor()
             msisdn = cur.execute('''SELECT msisdn FROM imsi_msisdn
                     WHERE imsi=?''', [IMSI]).fetchall()
             con.close()
-            return msisdn[0][0], 200
+            if msisdn[0][0]:
```

Comment:
Possible index error, the index is accessed before the check takes place. Causing an index error if index cannot be reached.

https://stackoverflow.com/a/29715530